src/context/SyntheticsStateContext/selectors/tradeboxSelectors/index.ts

```diff
@@ -774,3 +774,12 @@ export const selectTradeboxLeverageSliderMarks = createSelector((q) => {
   const maxAllowedLeverage = q(selectTradeboxMaxLeverage);
   return getTradeboxLeverageSliderMarks(maxAllowedLeverage);
 });
+
+export const selectTradeboxMarketsOrderMap = createSelector((q) => {
+  const { sortedAllMarkets } = q(selectTradeboxAvailableTokensOptions);
+
+  return sortedAllMarkets.reduce((acc, market, idx) => {
```

Comment:
I am not sure but lets try to keep keys as addresses where possible

beware that there are native tokens and non-native tokens so be careful when using a.indexToken and indexTokenAddress

Reply:
agree, fixed